.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: java
 jdk:
-  - openjdk7
+  - oraclejdk8
```

Comment:
we may have discussed this already, but do we really want to build using Java 8 given that we are targeting Java 7 for IJ < 2016?